client/packages/common/src/intl/locales/en/common.json

```diff
@@ -1133,6 +1133,7 @@
   "label.variants": "Variants",
   "label.ven": "VEN",
   "label.verified": "Verified",
+  "label.cancel": "Cancel",
```

Comment:
We have a button with "Cancel" text, but I figured these could actually be different words in other languages.

Reply:
this is a button too though right? Maybe label.mark_invoice_as_cancelled or something to give more context if it's needed for the translation?

Reply:
Okay, I updated both the key and the text to make it more explicit:

![Screenshot 2025-02-17 at 11 45 35 AM](https://github.com/user-attachments/assets/57db5b23-a748-4290-a1e3-b87c80d39ae5)

19c86ea798